src/Utils.test.js

```diff
@@ -37,3 +42,30 @@ test('Test extend', () => {
     expect(extend({ a: 0 }, {b: 1}, { c: 2})).toEqual({ a: 0, b: 1, c: 2});
     expect(extend({ foo: { bar: { baz: true }}}, { foo: { new: true, bar: { new: true, baz: false}}})).toEqual({ foo: { new: true, bar: { new: true, baz: false}}});
 });
+
+test('Test focusByQuerySelector', () => {
+    // Test fixture with three focusable elements.
```

Comment:
Let's add expect.assertions to make sure this test is checking expects.

Reply:
Done.